tests/taskcluster/bin/raptor_graph

```diff
@@ -30,18 +30,18 @@ RAPTOR_WORKER_TYPE = 'raptor';
 const TEST_ITERATIONS = 6;
 
 // FirefoxOS apps to include in the launch test
-const LAUNCH_TEST_APPS = [{ gaia_name: "calendar", th_name: "Calendar", th_prefix: "ca" },
-                          { gaia_name: "clock", th_name: "Clock", th_prefix: "cl" },
-                          { gaia_name: "communications/contacts", th_name: "Contacts", th_prefix: "co" },
-                          { gaia_name: "communications/dialer", th_name: "Dialer", th_prefix: "di" },
-                          { gaia_name: "email", th_name: "Email", th_prefix: "em" },
-                          { gaia_name: "fm", th_name: "FM", th_prefix: "fm" },
-                          { gaia_name: "gallery", th_name: "Gallery", th_prefix: "ga" },
-                          { gaia_name: "music", th_name: "Music", th_prefix: "mu" },
-                          { gaia_name: "settings", th_name: "Settings", th_prefix: "se" },
-                          { gaia_name: "sms", th_name: "SMS", th_prefix: "sm" },
-                          { gaia_name: "video", th_name: "Video", th_prefix: "vi" },
-                          { gaia_name: "test-startup-limit", th_name: "Test-startup", th_prefix: "te" }];
+const LAUNCH_TEST_APPS = [{ gaia_name: "calendar", th_name: "Calendar", th_test_prefix: "ca", th_result: "Cal" },
```

Comment:
So is there any confusion to having a symbol for the tests being something like "ca1" but the result is "Cal"?  Would it help to have both have the prefix of "cal"